lib/inspec/formatters/base.rb

```diff
@@ -68,6 +68,11 @@ def stop(notification)
       run_data[:platform] = {
         name: platform(:name),
         release: platform(:release),
+        network: platform(:network),
```

Comment:
As commented on #3869, I really think this data should come from something like an Ohai resource (see #3886). You can access resources even in places like this by using the Resource Registry.